### PR TITLE
lectorate 6, docs folder

### PR DESCRIPTION
**Pull request by abeggchr**

### CODE REVIEW

File: articles/rules-are-dangerous.md [hunk 1]
```diff
@@ -2,45 +2,45 @@
 authorName: Rolf Bruderer
 authorGithubUsername: bruderol
 issue: 31
-title: Rules are Dangerous
+title: Rules are dangerous
 ---
 # {{page.title}}
 
-Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us as engineers. As such we are used to see the world in rigid rules, true and false, if and else. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not bad in the first place. But we all must be very careful that we do not fall into the trap of just following such rules blindly without thinking well about consequences. 
+Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us, as engineers. As such we are used to looking at the world in terms of rigid rules, true and false, if and otherwise. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not a bad thing in itself. But we all must be very careful that we do not fall into the trap of just following such rules blindly without giving careful thought to consequences.
```
Comment (bruderol) [inline articles/rules-are-dangerous.md]: Please keep "true and false, if and else" - its a reference to technical programming words as usually used by programmers

File: articles/rules-are-dangerous.md [hunk 2]
```diff
@@ -2,45 +2,45 @@
...
+Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us, as engineers. As such we are used to looking at the world in terms of rigid rules, true and false, if and otherwise. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not a bad thing in itself. But we all must be very careful that we do not fall into the trap of just following such rules blindly without giving careful thought to consequences. 
 
-## Examples of Rules from Software Projects
+## Examples of rules from software projects
 
-Here are some examples of rules in Software Engineering that I have seen to cause troubles in projects:
+Here are some examples of rules in software engineering that I know from experience can cause troubles in projects:
```
Comment (bruderol) [inline articles/rules-are-dangerous.md]: The correction sounds too harsh - i would never say it like so - this sounds as if this rules always cause troubles ... and this is not about a fishy thing as "my experience" but about "what I have seen in projects" - so concrete observations - therefore I like my old formulation better. I do not like the formulation "I know" - I didnt want to say that I know this - i wanted to say that i have seen them to cause troubles. To me thus is not the same.

File: articles/rules-are-dangerous.md [hunk 3]
```diff
@@ -2,45 +2,45 @@
...
+* The dependency inversion principle says “Abstractions should not depend on details. Details should depend on abstractions". I have seen projects that tried to follow this principle very strictly and everywhere in their software. They therefore totally avoided directly using any concrete class as much as possible. As a result, there were a lot of interfaces with only one implementation class each and many factories to create instances of those classes. This resulted in a code base that is much more complex than needed. It makes the software difficult to understand and maintain without bringing any benefit of much- needed flexibility into that software.
```
Comment (bruderol) [inline articles/rules-are-dangerous.md]: What is "much- needed" - this looks weird, what I meant was "no benefit of flexibility that is at all needed" (Deutsch: keine Flexibilität die tatsächlich gebraucht wird)

File: articles/rules-are-dangerous.md [hunk 4]
```diff
@@ -2,45 +2,45 @@
...
 
-Please, do not get me wrong: I like the Agile Manifesto, the SOLID Principles, Clean Code, Martin Fowler in general and even his article about the Page Object Pattern. Those are all practices I appreciate a lot and that influenced my way of thinking and working. But still all these things also come with the danger of undesired side-effects, if just following them blindly in a dogmatic way.
+Please, do not get me wrong: I like the Agile Manifesto, the SOLID Principles, Clean Code, Martin Fowler in general, and even his article about the Page Object Pattern. Those are all practices that I appreciate a great deal and that have influenced my way of thinking and working. But, nevertheless, all these things also come with the danger of undesired side-effects, if they are just followed blindly in a dogmatic manner.
```
Comment (bruderol) [inline articles/rules-are-dangerous.md]: Wording "great deal" sounds strange to me. What is wrong with simply "appreciate a lot" or "appreciate much" ??
Rest is okay to me.

File: articles/rules-are-dangerous.md [hunk 5]
```diff
@@ -2,45 +2,45 @@
...
-* Note down not just the rules you use in your project but also the arguments why a rule should be applied. 
-* Be open to discuss and reexplain rules in your projects with your team members and to adjust the rules to changes in the context (e.g. growing software, growing team, shorter release cycles, different team members, new ideas, etc.)
-* Be critical to existing rules, but also respect and try to understand them in a first place.
+* We must all be always willing to explain ourselves, why we do something in a particular way and what the arguments are for applying a rule in a particular context. Something like “just because our process manual says so” or “because I’ve read it in that article” is usually not a good answer.
+* Don’t try to replace critical thinking and adaptive methods with exact and rigid rules that people must follow strictly. 
+* Do not read my recommendations here in a dogmatic way. Doing so would be to conclude: “Rolf said rules are bad and harmful, so let’s trash all the rules in our project and not follow the scrum guide, our Definition of Done, Clean Code and similar things anymore”. This was not my message at all!
```
Comment (bruderol) [inline articles/rules-are-dangerous.md]: I learnt from @peitor that "Scrum" should always be written with first letter in big - like God ;-)  maybe even "Scrum Guide"??

Comment (abeggchr) [reply articles/rules-are-dangerous.md]: done https://github.com/Zuehlke/fifty-shades/commit/5a689473f22bcb2bba1abc86d809c5e37268e6eb

File: articles/rules-are-dangerous.md (hunk 1 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: ok

File: articles/rules-are-dangerous.md (hunk 2 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: ok

File: articles/rules-are-dangerous.md (hunk 3 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: ok

File: articles/rules-are-dangerous.md (hunk 4 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: I would trust the expert here and leave it as corrected (a great deal).

File: articles/rules-are-dangerous.md (hunk 1 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: https://github.com/Zuehlke/fifty-shades/commit/36b464815149809af537129d854954e1e795eb1d

File: articles/rules-are-dangerous.md (hunk 2 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: https://github.com/Zuehlke/fifty-shades/commit/36b464815149809af537129d854954e1e795eb1d

File: articles/rules-are-dangerous.md (hunk 3 above)
Comment (abeggchr) [reply articles/rules-are-dangerous.md]: https://github.com/Zuehlke/fifty-shades/commit/36b464815149809af537129d854954e1e795eb1d

**Comment by bruderol** [comment]
Thanks @abeggchr ! Awesome!

**Pull request merged**